docs/DeveloperGuide.md

```diff
@@ -279,7 +279,7 @@ We help _busy working professionals_ manage their large list of contacts by prov
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
 | Priority | As a …​              | I want to …​                                                     | So that I can…​                                                                    |
-| -------- | ----------------------- | ------------------------------------------------------------------- | ------------------------------------------------------------------------------------- |
+| -------- |-------------------------|---------------------------------------------------------------------|---------------------------------------------------------------------------------------|
```

Comment:
Does this changes work for the website?

Reply:
I won't be able to test until it is deployed. Will revert just in case.